ocs_ci/ocs/resources/pvc.py

```diff
@@ -104,7 +104,7 @@ def get_all_pvcs(namespace=None):
     return out
 
 
-def create_multiple_pvc(number_of_pvc=1, pvc_data=None):
+def create_multiple_pvc(number_of_pvc=1, **pvc_data):
```

Comment:
This change is risky as you changing the function signature.
I'd say it is a good option to have that function with the option not to pass pvc_data.
you can pass it in your call